src/App.js

```diff
@@ -6,7 +6,7 @@ import categories from './components/common/categories'
 import InitialData from './components/common/InitialData'
 import ModalForm from './components/Form/ModalForm'
 import TimeRemaining from './components/TimeRemaining'
-import { loadActivities, save } from './utils'
+import { loadActivities, save } from './components/common/utils'
```

Comment:
I think `utils.js` was fine, where it was before. It has no dependency on any of your components.